beetsplug/mpdstats.py

```diff
@@ -53,6 +53,9 @@ def __init__(self, log):
         self.music_directory = (
             mpd_config['music_directory'].as_str())
 
+        self.strip_path = (
+            mpd_config['strip_path'].as_str())
```

Comment:
This can probably fit one one line (and therefore not need the somewhat odd-looking parentheses), right?

Reply:
Done, moved to one line